src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/cas1/Cas1BookingEmailServiceTest.kt

```diff
@@ -364,7 +369,7 @@ class Cas1BookingEmailServiceTest {
       val (application, booking) = createApplicationAndBooking(
         applicant,
         premises,
-        apArea = ApAreaEntityFactory().withEmailAddress(null).produce(),
+        apArea = ApAreaEntityFactory().produce(),
         arrivalDate = LocalDate.of(2023, 2, 1),
```

Comment:
ideally we'd define a cruManagementArea here with a null email address

Reply:
Added this below.